website/docs/ayon_launcher_artist_basic.md

```diff
@@ -0,0 +1,190 @@
+---
+id: ayon_launcher_artist_basic
+title: Basic Usage
+sidebar_label: Basic Usage
+description: AYON Launcher
+toc_max_heading_level: 5
+---
+
+:::info
+This page explains the basic usage of AYON Launcher, including the Tray Menu and how to use different actions.
+:::
+
+## Intro
+
+<div class="row">
+<div class="col">
+
+Once you launch the AYON Launcher, the AYON icon should appear in your system tray.
+
+</div>
+<div class="col">
+
+![A screenshot of the Windows System Tray showing the AYON icon](assets/artist_systray.png)
+
+</div>
+</div>
+
+:::tip Launcher Icon Color Code
+
+The AYON icon can appear in different colors, indicating the [pipeline mode](admin_server_bundles_and_addons.md#pipeline-modes) in which AYON was launched.
+- <span style={{color:'#23E0A9'}}>⬤</span> : AYON is in <span style={{color:'#1c2026',backgroundColor:'#23E0A9', borderRadius: '4px', padding: '2px 4px'}}>Production</span> mode.
+- <span style={{color:'#ff858b'}}>⬤</span> : AYON is in <span style={{color:'#1c2026', backgroundColor:'#ff858b', borderRadius: '4px', padding: '2px 4px'}}>Staging</span> mode.
+- <span style={{color:'#e2e2e3'}}>⬤</span> : AYON is in <span style={{color:'#1c2026',backgroundColor:'#e2e2e3', borderRadius: '4px', padding: '2px 4px'}}>Development</span> mode.
+
+The default executable starts AYON Launcher in <span style={{color:'#1c2026',backgroundColor:'#23E0A9', borderRadius: '4px', padding: '2px 4px'}}>Production</span> mode, which is the stable mode for daily work.
+
+For more info about running the launcher in different modes, see [Launch in Different Pipeline Modes](ayon_launcher_artist_advanced.md#launch-in-different-pipeline-modes).
+
+:::
+
+## Tray Menu
```

Comment:
Don't know if is relevant, but tray and all tools in tray are NOT defined by AYON launcher, but by ayon-core.